modular_skyrat/modules/SEVA_suit/suit_voucher.dm

```diff
@@ -0,0 +1,25 @@
+//the suit voucher
+/obj/item/suit_voucher
+	name = "suit voucher"
+	desc = "A token to redeem a new suit. Use it on a mining equipment vendor."
+	icon = 'icons/obj/mining.dmi'
+	icon_state = "mining_voucher"
+	w_class = WEIGHT_CLASS_TINY
+
+
+//Code to redeem new items at the mining vendor using the suit voucher
+//More items can be added in the lists and in the if statement.
+/obj/machinery/mineral/equipment_vendor/proc/RedeemSVoucher(obj/item/suit_voucher/voucher, mob/redeemer)
+	var/items = list("SEVA suit" = image(icon = 'modular_skyrat/modules/SEVA_suit/icons/seva_suit.dmi', icon_state = "seva"))
```

Comment:
Seems just a tad redundant for only having one suit. But it does mean we can add more in future I suppose.